src/chart/cartesianChart.js

```diff
@@ -19,32 +19,53 @@ export default function(xScale, yScale) {
         },
         yLabel = '',
         xLabel = '',
+        xBaseline = noop,
```

Comment:
> Are you using noop rather than null because it's being run through d3.functor

Yes, that was my original thinking, but I've just realised, the default value isn't functor-ed, so `null` works just fine.